mirgecom/simutil.py

```diff
@@ -177,7 +176,11 @@ def write_visfile(discr, io_fields, visualizer, vizname,
 
 
 def allsync(local_values, comm=None, op=None):
```

Comment:
Kind of not loving the function name? Make an issue to rename to `allreduce`?

Reply:
When we rolled this, we preferred `allsync` over `allreduce` is already an MPI routine.

Reply:
It carries out a reduction. It should have "reduction" in the name IMO. "sync" kind of sounds like synchronization primitive. It could be called `global_reduction` or something.

Reply:
(#509) Added this to the housecleaning issue because it is used by all the drivers, and is a change unrelated to this PR.